tests/installer/installer_test.go

```diff
@@ -0,0 +1,68 @@
+package cos_test
+
+import (
+	"github.com/rancher-sandbox/cOS/tests/sut"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("cOS Installer tests", func() {
+	var s *sut.SUT
+
+	BeforeEach(func() {
+		s = sut.NewSUT()
+		s.EventuallyConnects()
+	})
+
+	AfterEach(func() {
+		if CurrentGinkgoTestDescription().Failed {
+			s.GatherAllLogs()
+		} else {
+			// Trash disk so we boot from iso after each test
+			s.EmptyDisk("/dev/sda")
```

Comment:
maybe we should empty the disk before each one? Otherwise it depends from the first one to be able to run successfully (and can be a bit confusing?)